controller/src/main/java/com/softserve/borda/config/jwt/JwtProvider.java

```diff
@@ -19,10 +19,8 @@
     private String jwtSecret;
 
     public String generateToken(String login) {
-        Date date = Date.from(LocalDate.now().plusDays(15).atStartOfDay(ZoneId.systemDefault()).toInstant());
         return Jwts.builder()
                 .setSubject(login)
-                .setExpiration(date)
                 .signWith(SignatureAlgorithm.HS512, jwtSecret)
                 .compact();
```

Comment:
An authorized user is always in status Online on our website.

Reply:
Must have been changed when JWTConverter was used(token was stored in database, so if it gets expired, user would not be able to log in). Fixed it